sdk/communication/azure-mgmt-communication/setup.py

```diff
@@ -84,7 +84,8 @@
         'msrest>=0.5.0',
         'msrestazure>=0.4.32,<2.0.0',
         'azure-common~=1.1',
-        "azure-core<2.0.0,>=1.2.2"
+        "azure-core<2.0.0,>=1.10.1",
+        'azure-mgmt-core>=1.2.0,<2.0.0',
```

Comment:
Why are we adding this dependency?

Reply:
Removing this dependency, it was an attempt to fix the dependency failure, but appears to be unnecessary